recipes/beautifulsoup4/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "4.5.3" %}
+{% set name = "beautifulsoup4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 937e0df0d699a1237646f38fd567f0c6
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - bs4
+
+about:
+  home: http://www.crummy.com/software/BeautifulSoup/index.html
```

Comment:
They seem to be using [1] on [PyPI]( https://pypi.python.org/pypi/beautifulsoup4/4.5.3 ). Though it redirects to [2], which seems the same as [3] proposed here.

1 - http://www.crummy.com/software/BeautifulSoup/bs4/
2 - https://www.crummy.com/software/BeautifulSoup/
3 - http://www.crummy.com/software/BeautifulSoup/index.html